README.md

```diff
@@ -1 +1,105 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+This lesson will cover how and why you would import JSON files.
+
+## Objectives
+
+1. Describe how to import JSON files using provided example: configurations, data, settings
+1. Describe why you'd need to have JSON files
+
```

Comment:
Think of objectives as filling in the rest of the sentence "Students will be able to..."

You'd really want the first objective to be "Import JSON files..."  because the goal is to have students actually be able to write the code to import JSON files